code/contact.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en"
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wo gibt's das Brot zu probieren?</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <a href="./">Willkommen!</a>
+</head>
+<body>
+    <h1>Kontakt</h1>
+    <p>Mein wöchentlicher Backtag ist freitags. Am Mittwoch vorher informiere ich meine Kunden über die geplante Produktion, damit sie ihr Wunschbrot bestellen können. So stelle sicher, dass ich nicht zu viel produziere.</p>
+    <p>Möchtest du einmal Brot probieren oder künftig bestellen? Sehr gerne nehme ich dich in meine Verteilerliste auf. Hinterlass mir doch einfach deine Kontaktdaten und du hörst bald von mir.</p>
+    <form>

```

Comment:
Add the action and method attributes to be able to see your data after submission:
```
<form
  action="http://httpbin.org/anything"
  method="POST">
```